prebuilts/etcd.toml

```diff
@@ -0,0 +1,29 @@
+id = "etcd"
+name = "ETCD"
+icon = "https://avatars.githubusercontent.com/u/41972792?s=200&v=4"
+description = "etcd is a distributed key-value store designed to securely store data across a cluster."
+
+[source]
+image = "bitnami/etcd:3.5.10"
+
+[[ports]]
+id = "client_requests"
+port = 2379
+type = "TCP"
+
+[[ports]]
+id = "peer_communication"
+port = 2380
+type = "TCP"
```

Comment:
if we run single instance etcd, we do not need to export this port.

Reply:
(but this is prebuilt perhaps it uses this port

Reply:
@yuaanlin told me that zeabur could only export only one port for each container, so I am not sure `2379` and `2380` could be both exported.

Reply:
If a prebuilt service has two tcp port, both can access from other services in the same environment. But if you want to connect it via public network (port forwarding), only one port can access.